oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/service/ServiceTraffic.java

```diff
@@ -93,14 +101,23 @@ public ServiceTraffic map2Data(final Map<String, Object> dbMap) {
             ServiceTraffic serviceTraffic = new ServiceTraffic();
             serviceTraffic.setName((String) dbMap.get(NAME));
             serviceTraffic.setNodeType(NodeType.valueOf(((Number) dbMap.get(NODE_TYPE)).intValue()));
+            serviceTraffic.setGroup((String) dbMap.get(GROUP));
             return serviceTraffic;
         }
 
         @Override
         public Map<String, Object> data2Map(final ServiceTraffic storageData) {
+            final String serviceName = storageData.getName();
+            if (NodeType.Normal.equals(storageData.getNodeType())) {
+                int groupIdx = serviceName.indexOf(DOUBLE_COLONS_SPLIT);
+                if (groupIdx > 0) {
+                    storageData.setGroup(serviceName.substring(0, groupIdx));
+                }
+            }
             Map<String, Object> map = new HashMap<>();
-            map.put(NAME, storageData.getName());
+            map.put(NAME, serviceName);
```

Comment:
need to keep group as a service name prefix?

Reply:
Yes, we don't remove it. As the name is a part of metrics key. Changing it will break many things.

Reply:
ok